test/integration/logs/benchmark/benchmark_test.go

```diff
@@ -50,11 +52,23 @@ func BenchmarkEncoding(b *testing.B) {
 			return nil
 		}
 		messages, stats, err := loadLog(path)
+		if err != nil {
+			if os.IsNotExist(err) {
+				// Ignore symlinks to missing files.
```

Comment:
Is this still needed as we removed symlinks?

Reply:
No, removed.